### ISSUE TITLE
Programmatic Access - Bot Framework Web Chat- Card Flight]: 'Alt ' text or role='none/presentation' is not defined for the for the images.

### ISSUE THREAD

**Issue by manojs5** [microsoft/BotFramework-WebChat#5351]
### Is it an issue related to Adaptive Cards?

No.

### User Experience: 
When <img> elements lack alt text, it can significantly hinder the accessibility of the content for these users. Screen readers rely on alt text to convey the meaning and context of images to users who cannot view them visually. Without alt text, screen reader users may miss important information or context provided by the images.
### Note:
User credentials should NOT be included in the bug.

### What browsers and screen readers do this issue affect?

Windows: Edge with Windows Narrator

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

ARIA roles

### What is the public URL for the website?

https://compulim.github.io/webchat-loader/

### How to reproduce the issue?

1. Open URL: https://compulim.github.io/webchat-loader/ and sign in using valid credentials.
2. Navigate to web chat page and select 4.18.1-main.20240927.ea7a875 (30/9/2024) version and then select mockBot link under present label and then select open web chat in new window link at bottom.
3. Navigate to white label experience chat page and type Card Flight word in type your message text box.
4. Navigate to the search results.
5. Verify the issue.

### What do you expect?

There should be alternate text or a role of none or presentation for the images.
### Ex: 
Ensures <img> elements have alternate text or a role of none or presentation

### What actually happened?

'Alt ' text or role='none/presentation' is not defined for the images.
### Observation: 
Screen reader (Narrator, nvda, jaws) announces as 'Unlabeled graphic to get missing image descriptions, open the context menu'.

### Do you have any screenshots or recordings to repro the issue?

![Image](https://github.com/user-attachments/assets/01555e64-34d6-4ffa-8406-c3745a1179a4)
https://github.com/user-attachments/assets/29b60239-2eb9-4b80-b0c9-620eff443392

### MAS reference

MAS 1.1.1 – Non-text Content

### WCAG reference

1.1.1 Non-text Content

### Issue: 
Ensures <img> elements have alternate text or a role of none or presentation (image-alt - [](https://accessibilityinsights.io/info-examples/web/image-alt))

### Target application: 
Web Chat - [](https://compulim.github.io/webchat-loader/webchat.html?v=4.18.1-main.20240927.ea7a875&p=ws&sr=westus&uid=r_zse43advci&cid=1XnMdXiUAvbDd0FEPSjCeW-us&dt=eyJhbGciOiJSUzI1NiIsImtpZCI6Ik5MZjUwanFheTN1R0VpQUpOV29EWmZzRnE4QSIsIng1dCI6Ik5MZjUwanFheTN1R0VpQUpOV29EWmZzRnE4QSIsInR5cCI6IkpXVCJ9.eyJib3QiOiJoYXdvLW1vY2tib3Q0LW1vY2stYm90LWJvdCIsInNpdGUiOiJwRVp0UExiaTd5TSIsImNvbnYiOiIxWG5NZFhpVUF2YkRkMEZFUFNqQ2VXLXVzIiwibmJmIjoxNzMwNzg3ODAwLCJleHAiOjE3MzA3OTE0MDAsImlzcyI6Imh0dHBzOi8vZGlyZWN0bGluZS5ib3RmcmFtZXdvcmsuY29tLyIsImF1ZCI6Imh0dHBzOi8vZGlyZWN0bGluZS5ib3RmcmFtZXdvcmsuY29tLyJ9.Wx37hBigPA99Nul7rfLF8NXGudjOif4s839AHDE7oFoXSvjLhpQrTkrKVlPdJngMkL8nR9ufOENpTZKDWkRlUz6BdSF-3E7Kn6-qPwo_9XMntddJYqrIcejX4kSuJgwrko2vzMz_khR5jgDqsZeGDWndE8YHZBLvua_40I4ozkr86KXml3_ZmyP7yGCYAJxlOHvhaZAAqwLZRJRXpY4RBT3B4qBTn5KTDbiehvxaTw7GTsqCUUo97UbNrmbEhuCnVPftLuPk-QsZmGfU9PkX7T1zLmo02p5tjbtbiNHMy-WRKzyBe0naN495r0-g_S15QkGAwc4_aHyR_6Qu2QD1IA&st=aad%23%2Fsubscriptions%2F51fbc48d-04f1-449a-b50c-b40618411b9f%2FresourceGroups%2Fhawo-mockbot4-rg%2Fproviders%2FMicrosoft.CognitiveServices%2Faccounts%2Fhawo-mockbot4-speech%23eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiIsIng1dCI6IjNQYUs0RWZ5Qk5RdTNDdGpZc2EzWW1oUTVFMCIsImtpZCI6IjNQYUs0RWZ5Qk5RdTNDdGpZc2EzWW1oUTVFMCJ9.eyJhdWQiOiJodHRwczovL2NvZ25pdGl2ZXNlcnZpY2VzLmF6dXJlLmNvbSIsImlzcyI6Imh0dHBzOi8vc3RzLndpbmRvd3MubmV0LzcyZjk4OGJmLTg2ZjEtNDFhZi05MWFiLTJkN2NkMDExZGI0Ny8iLCJpYXQiOjE3MzA3NzY4NTIsIm5iZiI6MTczMDc3Njg1MiwiZXhwIjoxNzMwODYzNTUyLCJhaW8iOiJrMkJnWUlpYUpEZHQ3WXoybHBUejY5VWVUL25WREFBPSIsImFwcGlkIjoiOTE4NGU1OTQtZWIwMi00YmU2LWExMTAtOWIyM2M3MTNlMmRmIiwiYXBwaWRhY3IiOiIyIiwiaWRwIjoiaHR0cHM6Ly9zdHMud2luZG93cy5uZXQvNzJmOTg4YmYtODZmMS00MWFmLTkxYWItMmQ3Y2QwMTFkYjQ3LyIsImlkdHlwIjoiYXBwIiwib2lkIjoiYmZjMzNkMGEtMjBiYS00YTE5LWI4ZTEtYWYxN2JmNDIxYWM5IiwicmgiOiIxLkFCb0F2NGo1Y3ZHR3IwR1JxeTE4MEJIYlI1QWlNWDNJS0R4SG9PMk9VM1NiYlcwYUFBQWFBQS4iLCJzdWIiOiJiZmMzM2QwYS0yMGJhLTRhMTktYjhlMS1hZjE3YmY0MjFhYzkiLCJ0aWQiOiI3MmY5ODhiZi04NmYxLTQxYWYtOTFhYi0yZDdjZDAxMWRiNDciLCJ1dGkiOiI3eWc0VkF1dlpFT3FqV3ZpdXZmQ0FBIiwidmVyIjoiMS4wIiwieG1zX2F6X3JpZCI6Ii9zdWJzY3JpcHRpb25zLzUxZmJjNDhkLTA0ZjEtNDQ5YS1iNTBjLWI0MDYxODQxMWI5Zi9yZXNvdXJjZWdyb3Vwcy9oYXdvLW1vY2tib3Q0LXJnL3Byb3ZpZGVycy9NaWNyb3NvZnQuQXBwL2NvbnRhaW5lckFwcHMvaGF3by1tb2NrYm90NC10b2tlbi1hcHAiLCJ4bXNfaWRyZWwiOiI3IDE2IiwieG1zX21pcmlkIjoiL3N1YnNjcmlwdGlvbnMvNTFmYmM0OGQtMDRmMS00NDlhLWI1MGMtYjQwNjE4NDExYjlmL3Jlc291cmNlZ3JvdXBzL2hhd28tbW9ja2JvdDQtcmcvcHJvdmlkZXJzL01pY3Jvc29mdC5NYW5hZ2VkSWRlbnRpdHkvdXNlckFzc2lnbmVkSWRlbnRpdGllcy9oYXdvLW1vY2tib3Q0LXNwZWVjaC1pZGVudGl0eSJ9.a2NEe6kHaZ49pbAG9ZU0nB29-eQ2JthoD_gIsrdjVdxgsYWNXzvW55DPuq1t6b5pb5Mbr7eJ978EkFAe6F-uQ_blyUaIlUzkP9XH9XRQRjuIzR19JrJk2TcYPCNyjP-BdVn5OpmZ-deSXZSrK4JfVjhhXghjmIO6l6GGfg6YSkfqrRJtciyVS4SomnOBsY_WV8ixZEpMscR8IvFEq-axSQMAgd7j8Vcc3ZFlESRDaOmNzq1VdmfF-_nwOlMFesVNJH13M_4UqSCtobh3MIE-qFTZgaR1pcKMl7MdDVfBUQh8s8KI-4S3xbmiYGQHgp5WkYJTGNOSy_qzzlr3myrvtw)

### Element path: 
div:nth-child(3) > img

### Snippet: 
<img class="ac-image" src="http://adaptivecards.io/content/airplane.png" style="min-width: 0px; width: 40px; max-height: 100%;">

### How to fix: 
Fix any of the following:

- Element does not have an alt attribute
- aria-label attribute does not exist or is empty
- aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
- Element has no title attribute
- Element's default semantics were not overridden with role="none" or role="presentation"

**Comment by manojs5** [comment]
#A11YAuto;#A11yMAS;#A11ySev2;#Accessibility;;#Benchmark;#BotFrameworkWebChat-Oct24;#HCL;#K4W;#MAS1.1.1;#Win11-Edge(Chromium);

**Comment by Kirank915** [comment]
@OEvgeny This issue is marked as external to Adaptive cards, could you please confirm if your team will fix the issue, or we need to share this issue to adaptive cards team?

**Comment by OEvgeny** [comment]
This is adaptive card authoring issue. Can be fixed by including an alt into the card description to avoid the issue. No further action is needed.

**Comment by Kirank915** [comment]
@OEvgeny could you please help us with service tree ID of the Adaptive cards team so we can route the bug as external to adaptive cards team.

**Comment by OEvgeny** [comment]
The issue is not in how adaptive cards handle this particular card, rather than in card definition. The card definition can be adjusted, so it doesn't break accessibility. This is something bot author is able to do, but for this particular case, we won't update the test bot (Mockbot) for now.
